src/workerd/util/autogate.c++

```diff
@@ -33,23 +34,22 @@ Autogate::Autogate(capnp::List<autogate::Autogate, capnp::Kind::STRUCT>::Reader
     // Parse the gate name into a AutogateKey.
     for (AutogateKey i = AutogateKey(0); i < AutogateKey::NumOfKeys; i = AutogateKey((int)i + 1)) {
       if (kj::str(i) == sliced) {
-        gates.insert(i, autogate.getEnabled());
-        break;
+        gates[(unsigned long)i] = true;
```

Comment:
So our plan here is to silently ignore unrecognized keys? The downside is obviously that it's worse for catching typos than logging a warning would be. Another possibility is that the key is meant for a different version of the runtime than the one you are using, which is also probably helpful to know. 

I guess the benefit is that we can remove a key entirely from the codebase without causing trouble. The fact that v8's `--experimental-XXX` flags trigger an error once the feature has been enabled globally is quite annoying because you end up needed to check the v8 version to calculate what flags to pass. A third option would be to keep a list of the removed flags somewhere so we can know if someone passed an old flag.

Perhaps add a comment under this like:
```C++
// Ignore unrecognized flag
// TODO: consider warning about this somehow
```

Reply:
Good point, a warning here would be good. I'll add one.